app/constants.js

```diff
@@ -20,6 +20,7 @@ export const INSTANCES_FOLDER = 'instances';
 export const SERVERS_PATH = path.join(DATAPATH, 'servers');
 export const INSTANCES_PATH = (store.has('settings') && store.get('settings').instancesPath) ? store.get('settings').instancesPath : path.join(DATAPATH, INSTANCES_FOLDER);
 export const PACKS_PATH = path.join(INSTANCES_PATH, 'packs');
+// TODO Move meta folder withinstances folder when changing where instances folder is stored in the settings.
```

Comment:
Does This Still need to be done?

Reply:
Didn't see a good way to use the store.get('settings').instancesPath to move this folder along with it unless it was inside it.

Would be good if this was stored where the rest of instances files are like assets, libraries, packs, and versions are since they have to do with them.
Moving them would require users to reinstall all forge versions since they would not be found in the new location.